src/app/hooks/useViewTracker/index.jsx

```diff
@@ -31,12 +28,30 @@ const useViewTracker = (props = {}) => {
     statsDestination,
   } = useContext(EventTrackingContext);
   const { service } = useContext(ServiceContext);
+  const initObserver = async () => {
+    if (typeof window.IntersectionObserver === 'undefined') {
+      // Polyfill IntersectionObserver, e.g. for IE11
+      await import('intersection-observer');
+    }
+    const callback = changes => {
+      const someElementsAreInView = changes.some(prop('isIntersecting'));
+
+      setIsInView(someElementsAreInView);
+    };
+    const options = {
+      threshold: [MIN_VIEWED_PERCENT],
+    };
+
+    observer.current = new IntersectionObserver(callback, options);
+  };
 
   useEffect(() => {
-    if (eventTrackingIsEnabled && inView && !timer.current) {
+    if (isInView && !timer.current) {
       timer.current = setTimeout(() => {
-        const shouldSendEvent = [
-          !viewSent,
+        const shouldSendEvent = [trackingIsEnabled, isInView, !eventSent].every(
```

Comment:
```suggestion
        const shouldSendEvent = [trackingIsEnabled, !eventSent].every(
```
I think the `isInView` check is redundant here as you can't initiate this callback unless you pass the check on line 49: https://github.com/bbc/simorgh/pull/9145/files#diff-7847911c4a756dfc4dce2205ea165af35dbdbb6c296083a0a02862757737cb92R49

Reply:
I think you're right but I need to have a think about this one because we are in a timeout here and I wonder if it's possible for `isInView` to mutate when the timeout is finshed

Reply:
I think you might be right @jroebu14 so:
- the timeout is initiated because the item being tracked is in view
- the required timeout elapses
- the item is no longer in view

Yeah your current cases catches that scenario, do you have a test for this? If you do, making the change I suggested would cause it fail and prove that my suggestion shouldn't be implemented.

Reply:
I think you were right before because if the `isInView` is ever false then the timeout is cleared on this line https://github.com/bbc/simorgh/pull/9145/files#diff-7847911c4a756dfc4dce2205ea165af35dbdbb6c296083a0a02862757737cb92R88